packages/uppy/index.js

```diff
@@ -22,6 +22,7 @@ exports.StatusBar = require('@uppy/status-bar')
 
 // Acquirers
 exports.Dropbox = require('@uppy/dropbox')
+exports.Box = require('@uppy/box')
```

Comment:
excluding box from global module for now as it will initially be released as beta

```suggestion
```